packages/ibm-products/package.json

```diff
@@ -54,52 +54,52 @@
     "upgrade-dependencies": "npm-check-updates -u --dep dev,peer,prod --color --reject '/(carbon|^react$|^react-dom$|^chalk$|^namor)/'"
   },
   "devDependencies": {
-    "@babel/cli": "^7.23.9",
-    "@babel/core": "^7.23.9",
+    "@babel/cli": "^7.24.5",
+    "@babel/core": "^7.24.5",
     "@babel/plugin-proposal-class-properties": "^7.17.12",
-    "@babel/plugin-proposal-export-default-from": "^7.17.12",
+    "@babel/plugin-proposal-export-default-from": "^7.24.1",
     "@babel/plugin-proposal-export-namespace-from": "^7.17.12",
```

Comment:
`@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-export-namespace-from` have been merged to the standard and are no longer proposals. The graduated packages can be used (sorry it won't let me make a suggestion across deleted lines):

- [@babel/plugin-transform-class-properties](https://www.npmjs.com/package/@babel/plugin-transform-class-properties)
- [@babel/plugin-transform-export-namespace-from](https://www.npmjs.com/package/@babel/plugin-transform-export-namespace-from)